appspec.yml

```diff
@@ -13,18 +13,18 @@ permissions:
 
 hooks:
   BeforeInstall:
-    - location: server_clear.sh
+    - location: scripts/server_clear.sh
```

Comment:
No need to scripts pre-path for our case because on buildspec.yml file we described it like that. All files under the scripts file will be included as separated file.